capstone-frontend/src/components/Browse.js

```diff
@@ -1,11 +1,10 @@
-import React, { Component } from 'react';
+import React from 'react';
+import BookSearchList from './BookSearchList';
 
-class Browse extends Component {
-  render() {
-    return (
-      <div> lets put search results here </div>
-    );
-  }
+const Browse = (props) => {
+  return (
+    <BookSearchList key={props.match.params.query} searchQuery={props.match.params.query}/>
```

Comment:
Glad we figured out how to assign key to force a re-render on new searches by the user, but would it make sense to remove the searchQuery prop and just reference it by the key (since they're the same value anyways)?

Reply:
I tried removing the searchQuery prop and using the key in the query to the backend but it just passes `undefined` for some reason. I'll leave it as searchQuery